webpack.playground.config.js

```diff
@@ -81,14 +87,24 @@ module.exports = {
           presets: [
             // preset-env is disabled for a better debugging experience.
             // It can be enabled if necessary to run playground on IE11
-            // '@babel/preset-env',
+            ...(process.env.IE11_COMPAT_MODE === 'true' ? [
+              [
+                '@babel/preset-env',
+                {
+                  targets: {
+                    ie: '11'
+                  }
+                }
+              ]
+            ] : []),
             '@babel/preset-typescript' // must run before preset-env: https://github.com/babel/babel/issues/12066
           ]
         }
       },
     ]
   },
   devServer: {
+    ...hotReloadOptions,
     host: HOST,
```

Comment:
Could you also set the `host` option of the webpack-dev-server to `0.0.0.0` in `IE11_COMPAT_MODE` since it's common to want to test in a local VM?